### ISSUE TITLE
error: RPC failed; HTTP 408 curl 18 HTTP/2 stream 7 was reset

### ISSUE THREAD

**Issue by nseidle** [dmnemec/copy_file_to_another_repo_action#79]
Thanks for this action!

My workflow has been working for ~2 years. Last week, this error appeared when I try to move a 2.1MB file to another repo. The issue and work-around are discussed [here](https://github.com/orgs/community/discussions/55820). 

tldr; This action needs the following added:

    git config --global http.version HTTP/1.1

I'm currently using [this fork](https://github.com/Jason2866/copy_file_to_another_repo_action/tree/http408_fix) that has the ~~fix~~ work-around in place.

PR #78 looks like it is attempting to fix the same issue.

**Comment by dobbelina** [comment]
@nseidle 
My fork has the fix in place + [git-lfs support](https://github.com/dobbelina/copy_file_to_another_repo_action)

**Comment by Sanni-Damilola** [comment]
> Thanks for this action!
> 
> My workflow has been working for ~2 years. Last week, this error appeared when I try to move a 2.1MB file to another repo. The issue and work-around are discussed [here](https://github.com/orgs/community/discussions/55820).
> 
> tldr; This action needs the following added:
> 
> ```
> git config --global http.version HTTP/1.1
> ```
> 
> I'm currently using [this fork](https://github.com/Jason2866/copy_file_to_another_repo_action/tree/http408_fix) that has the ~fix~ work-around in place.
> 
> PR #78 looks like it is attempting to fix the same issue.

git config --global http.version HTTP/1.1
